proposals/0452-integer-generic-parameters.md

```diff
@@ -3,9 +3,9 @@
 * Proposal: [SE-0452](0452-integer-generic-parameters.md)
 * Authors: [Alejandro Alonso](https://github.com/Azoy), [Joe Groff](https://github.com/jckarter)
 * Review Manager: [Ben Cohen](https://github.com/airspeedswift)
-* Status: **Active Review (Nov 5-19 2024)**
+* Status: **Active review (November 5-19 2024)**
```

Comment:
Isn't this change needed?

```suggestion
* Status: **Active review (November 5...19, 2024)**
```

Reply:
Both formats have been used historically and are acceptable. Both will be extracted correctly by the metadata extractor tool.